rail_scripts/rail-preprocess-parquet

```diff
@@ -63,7 +63,7 @@ Configuration = namedtuple('Configuration',
                            ('input', 'output_dir', 'flux_template',
                             'flux_error_template', 'mag_template',
                             'mag_error_template', 'rows', 'flux',
-                            'output_template', 'ra','dec'))
+                            'output_template', 'ra', 'dec', 'objectId'))
```

Comment:
Inconsistent coding style: please use underscore notation for the configuration attributes: `object_id`.